commands/instances.go

```diff
@@ -702,3 +707,151 @@ func LoadSketch(ctx context.Context, req *rpc.LoadSketchReq) (*rpc.LoadSketchRes
 		AdditionalFiles:  additionalFiles,
 	}, nil
 }
+
+// ArchiveSketch FIXMEDOC
+func ArchiveSketch(ctx context.Context, req *rpc.ArchiveSketchReq) (*rpc.ArchiveSketchResp, error) {
```

Comment:
Other commands are in their corresponding `commands/xxxx` subfolder that mimics the `cli/xxxx`, In this case this command should be inside `commands/archive`.